packages/app/src/embed/theme/index.js

```diff
@@ -0,0 +1,20 @@
+import dot from 'dot-object';
+import applicationTheme from '@codesandbox/common/lib/theme';
+import codesandboxBlack from '@codesandbox/common/lib/themes/codesandbox-nu.json';
+import tokens from './tokens';
+
+// merge vscode colors into tokens
+Object.assign(tokens.colors, dot.object({ ...codesandboxBlack.colors }));
```

Comment:
🔖 i should leave a fat comment here explaining this voodoo merge and why I had to do it